HTML-37/tapnumber.js

```diff
@@ -0,0 +1,72 @@
+let textColorRandomNumber = randomNumberFunction(0,6,6);
```

Comment:
Variable name can be more intuitive.
Logic can be simpler if you choose just between 0 and 1 or true and false, for example:
const doTap=Math.random() > 0.5;